Accounts.API/appsettings.json

```diff
@@ -7,9 +7,13 @@
     },
     "AllowedHosts": "*",
 
-    // Add the necessary Kafka configs:
     "Kafka": {
         "BootstrapServers": "localhost:9092",
         "GroupId": "groupId"
+    },
+
+    "ConnectionStrings": {
+        "DefaultConnection": "Host=localhost;Port=5432;Username=postgres;Password=postgres;Database=postgres"
```

Comment:
Refactoring skal med i tankerne, for vi må ikke have vores .env variabler stående i åbne dokumenter på github da det er security risk - vi må fælles kigge ind i at få hentet dem fra vores env variabler

Reply:
Yes er skrevet ned i mine noter at vi lige skal kigge på det